### PR TITLE
Make translations work for modules

### PR DESCRIPTION
**Pull request by FransMeerhoff**
- Some code changes in LanguageModel and DynamicModule needed.
- Project files changes: make translation files for each single module.

### CODE REVIEW

File: Desktop/modules/dynamicmodule.cpp [hunk 1]
```diff
@@ -272,9 +281,10 @@ Description * DynamicModule::instantiateDescriptionQml(QString descriptionTxt, Q
 	return description;
 }
 
-void DynamicModule::loadDescriptionQml(QString descriptionTxt)
+
+void DynamicModule::loadDescriptionQml(QString descriptionTxt, QUrl & url)
```
Comment (boutinb) [inline Desktop/modules/dynamicmodule.cpp]: You could also set the description as const QString&

File: Desktop/modules/dynamicmodules.cpp [hunk 2]
```diff
@@ -111,21 +111,24 @@ bool DynamicModules::initializeModule(Modules::DynamicModule * module)
...
+			emit loadModuleTranslationFile(module);
+			
 			module->initialize();
 		}
 
 		if(!wasAddedAlready)
 		{
 			emit dynamicModuleAdded(module);
-			emit loadModuleTranslationFile(module);
+			//emit loadModuleTranslationFile(module);
```
Comment (boutinb) [inline Desktop/modules/dynamicmodules.cpp]: Why is it commented out? If it is really not necessary, just remove it.

**Review by boutinb — CHANGES_REQUESTED** [review]
Don't forget that we have also a chinese translation.

File: Desktop/modules/dynamicmodules.cpp (hunk 2 above)
Comment (FransMeerhoff) [reply Desktop/modules/dynamicmodules.cpp]: The emit was moved to the  ! initialized but was unnecessary. I've put it back

Comment (boutinb) [reply Desktop/modules/dynamicmodules.cpp]: So suppress this line of code. I don't like to see commented out codes when it's not explained

Comment (JorisGoosen) [reply Desktop/modules/dynamicmodules.cpp]: Double that ;)

**Pull request merged**